app/about_us/our_crew/page.js

```diff
@@ -0,0 +1,114 @@
+import styles from "./page.module.css";
+
+const OurCrew = () => {
+  // TASK - React 1 week 1
```

Comment:
These comments can be removed for the next HWs.

Reply:
Ok Noted